styleguide/progress.scss

```diff
@@ -0,0 +1,46 @@
+@import 'colors';
+@import 'typography';
+
+// progress bars
+.kiln-progress-wrapper {
+  bottom: 100%;
```

Comment:
it's absolutely positioned, and above everything else (or rather, it lives in the same place as the toolbar)